api/src/YoutubeClient.h

```diff
@@ -20,6 +20,8 @@ class VideosResult : public APIResult {
 
     uint64_t GetViewers() const;
 
+    bool IsAgeRestricted() const;
```

Comment:
maybe just call it `IsNSFW`? the angelthump scraper could implement this too...

Reply:
Done